File: day 0101/classwork/classwork.js
const numbers = [1,2,3,4,5,6,7,8,9,10];

numbers.forEach(curValue => {
    curValue % 2 === 0? console.log("Number is Even") :
    console.log("Number is Odd");
})

const alphabet = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"];

const symbols = alphabet.map((curValue, index) => `${curValue} - ${index + 1}`);

console.log(symbols);

const people = [
  { name: "Alice", age: 30, occupation: "Engineer" },
  { name: "Bob", age: 25, occupation: "Designer" },
  { name: "Charlie", age: 35, occupation: "Teacher" },
  { name: "David", age: 28, occupation: "Developer" },
  { name: "Eva", age: 32, occupation: "Manager" }
];

const filteredPeopleArray = people.filter(curValue => curValue.age >= 30);

console.log(filteredPeopleArray);

File: day 0019/classwork/classwork.py
numbers = [1, 7, 9, 21, 4, 88, 15, 6, 333, 10]

even_sum = 0
odd_sum = 0

index = 0
while index < len(numbers):
    if numbers[index] % 2 == 0:
        even_sum += numbers[index]
    else: 
        odd_sum += numbers[index]
    index += 1

sum = even_sum = odd_sum 

print("უდიდესი ჯამი არის:",sum)